release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -103,11 +103,29 @@ By default, the following network connections are checked:
 
 {product-title} 4.6 introduces the following notable technical changes.
 
+[discrete]
 [id="ocp-4-6-ovn-k8s-default-cni-np-uses-ovs-on-cluster-nodes"]
 ==== OVN-Kubernetes default CNI network provider now uses OVS installed on cluster nodes
 
 The OVN-Kubernetes default Container Network Interface (CNI) network provider now uses the Open vSwitch (OVS) version installed on the cluster nodes. Previously, OVS ran in a container on each node, managed by a DaemonSet. Using the host OVS eliminates any possible downtime from upgrading the containerized version of OVS.
 
+[discrete]
+[id="ocp-4-6-warnings-when-using-deprecated-apis"]
+==== Warnings when using deprecated APIs
+
+Warnings are now visible in `client-go` and `oc/kubectl` on every invocation
```

Comment:
I think you can remove "kubectl" here - we usually just show "oc" throughout our docs.